<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zork</class>
 <widget class="QMainWindow" name="Zork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zork</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(98, 98, 98);</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>581</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="Main">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QTextEdit" name="Console">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>261</width>
        <height>391</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Up">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>120</y>
        <width>80</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);
</string>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Left">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>80</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Right">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>170</y>
        <width>80</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Down">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>220</y>
        <width>80</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="CurrentRoom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Search">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>170</y>
        <width>80</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(143, 143, 143);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="Lock">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Enemy">
     <attribute name="title">
      <string>Enemy</string>
     </attribute>
     <widget class="QTextBrowser" name="EnemyConsole">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>541</width>
        <height>192</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 4);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="HealthBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>541</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 4);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="HealthBarLabel">
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>210</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>22</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Health Bar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Battle">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string>Battle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Attack">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>390</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string>Attack</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Shop">
     <attribute name="title">
      <string>Shop</string>
     </attribute>
     <widget class="QComboBox" name="Keys">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>531</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(4, 255, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentText">
       <string>Keys</string>
      </property>
      <item>
       <property name="text">
        <string>Keys</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Silver</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gold</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SectionB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SectionC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SectionD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Final</string>
       </property>
      </item>
     </widget>
     <widget class="QTextBrowser" name="KeyConsole">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>291</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="KeyBuy">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string>Buy</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Weapons">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>531</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
selection-color: rgb(4, 255, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Weapons</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Basic Sword</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Stone Sword</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Silver Sword</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Crystal Sword</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Laser Sword</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fire Sword</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>God Sword</string>
       </property>
      </item>
     </widget>
     <widget class="QTextBrowser" name="WeaponConsole">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WeaponBuy">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>310</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string>Buy</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Balance">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 8);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="KeyTab">
     <attribute name="title">
      <string>Keys</string>
     </attribute>
     <widget class="QListWidget" name="KeyInventory">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>531</width>
        <height>371</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="KeyInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>531</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="WeaponsTab">
     <attribute name="title">
      <string>Weapons</string>
     </attribute>
     <widget class="QListWidget" name="WeaponsInventory">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="WeaponsInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>311</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Upgrade">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string>Upgrade</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Balance2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(26, 255, 0);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Equip">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>410</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string>Equip</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Map">
     <attribute name="title">
      <string>Map</string>
     </attribute>
     <widget class="QLabel" name="MapPic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>471</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Info">
     <attribute name="title">
      <string>Info</string>
     </attribute>
     <widget class="QTextBrowser" name="Information">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>531</width>
        <height>471</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
